### ISSUE TITLE
Midterm Project Review by Anilkumar Vadali

### ISSUE THREAD

**Issue by arv42** [vanshikabansal/ORIE-4741#10]
This project delves into a very interesting project that focuses on predicting the winner of a certain horse race based on many different factors and features, including race conditions, horse characteristics, and previous race results. Specifically, the group hoped to develop a model that would be able to find the expected position of a certain horse in a race and moreover, tried to expand on the provided project by hoping to create an optimal long-term strategy that would result in a profit. The data they utilized was from  horse races in Hong Kong from 2014 to 2016. Overall, I think that this project is useful and helpful to betters, jockeys, and owners involved in horse races hoping to understand how to be successful in both betting and racing in certain types of races. 

There were many things I liked throughout the your midterm project. One thing that I believe was well done was considering overfitting and underfitting of your model and handling those cases appropriately. As a group, you went into a lot of detail on why certain features and methods were utilized and how they reduce the chances of overfitting and underfitting and I think that help clear a lot of the reasoning behind your choices. Another part of the report that I really liked was the data visualization that was done on the initial data. Specifically, the correlation plot gives a lot of information on how certain features relate to each other and how those relations could be used in your models. Finally, I think that developing a long-term strategy for betting profits is a great idea and I think that moving forward this will help exemplify the results and the success of your model in the end. 

Additionally, I think that there are some improvements to your process that you may consider. One assumption that was made was that each race is held in the same conditions for each horse.; as a result, it was mentioned that many of the other variables such as track condition, race track, etc. were dropped. I think that these types of features will definitely impact the performances and positions of horses, especially since some horses may perform better in some conditions than in others. Some other features that were deleted like time of day may also affect a certain horse's performance as well. By  not considering these types of features, the results may exaggerate or deflate a horse's actual chance of winning or finishing in a certain place. Also, as others have mentioned, I believe that the validation techniques described within the report were a bit confusing. I think that the reasoning behind the validation and test set sizes could be explained a bit more clearer and explored more. I believe that utilizing a larger train set that is (3/4) and (4/5) and a test set of (1/4) or (1/5) would be more beneficial in developing a good model. As you mentioned, k-fold cross validation would also help in this situation as well. Finally, I think that using some of the features besides numerical features could also help improve the model as well. You could handle categorical features by using some of the methods we have learned in class (maybe one-hot encoding or converting values into unique ids). 

Overall I think that this report was very well done and I think the group has done some great work so far!
